v2/pkg/protocols/common/helpers/eventcreator/eventcreator.go

```diff
@@ -16,6 +18,13 @@ func CreateEvent(request protocols.Request, outputEvent output.InternalEvent, is
 func CreateEventWithAdditionalOptions(request protocols.Request, outputEvent output.InternalEvent, isResponseDebug bool,
 	addAdditionalOptions func(internalWrappedEvent *output.InternalWrappedEvent)) *output.InternalWrappedEvent {
 	event := &output.InternalWrappedEvent{InternalEvent: outputEvent}
+
+	// Dump response variables if ran in debug mode
+	if isResponseDebug {
```

Comment:
Normally these `if`s are redundant and they should disappear in favor of `MsgFunc` (available in the latest `gologger`). But this we can also tackle in #1166

Reply:
Will be tackled in #1166